app/controllers/concerns/session_management.rb

```diff
@@ -0,0 +1,6 @@
+module SessionManagement
+  def clear_claim_session
+    session.delete(:slugs)
```

Comment:
something i noticed too, i think we want to get rid of this in future when we get a moment as we currently track all previous slugs and instead should be able to work out where we came from